### PR TITLE
colors2

### PR DESCRIPTION
**Pull request by jonborchardt**
This converts exitsting code to use new colors2 object

### CODE REVIEW

File: docs/src/modules/components/AppFrame.js [hunk 1]
```diff
@@ -12,19 +12,16 @@ import IconButton from '@mui/material/IconButton';
 import SvgHamburgerMenu from 'docs/src/icons/SvgHamburgerMenu';
 import Tooltip from '@mui/material/Tooltip';
 import Box from '@mui/material/Box';
-import SettingsIcon from '@mui/icons-material/SettingsOutlined';
```
Comment (jonborchardt) [inline docs/src/modules/components/AppFrame.js:15]: sorry, unrelated to this pr...
just removing crap int he header we dont use

File: docs/src/modules/components/AppLayoutDocs.js [hunk 2]
```diff
@@ -59,26 +58,9 @@ const StyledAppContainer = styled(AppContainer, {
   };
 });
 
-const ActionsDiv = styled('div')(({ theme }) => ({
```
Comment (jonborchardt) [inline docs/src/modules/components/AppLayoutDocs.js:62]: sorry, unrelated to this pr...
just removing crap int he header we dont use

File: packages/markdown/parseMarkdown.js [hunk 3]
```diff
@@ -263,9 +263,6 @@ function createRender(context) {
         `<a aria-labelledby="${hash}" class="anchor-link" href="#${hash}" tabindex="-1">`,
         '<svg><use xlink:href="#anchor-link-icon" /></svg>',
         '</a>',
-        `<button title="Post a comment" class="comment-link" data-feedback-hash="${hash}">`,
```
Comment (jonborchardt) [inline packages/markdown/parseMarkdown.js:266]: sorry, unrelated to this pr...
just removing crap int he header we dont use

File: docs/data/material/varnish/colors/LightCategoricalColorValues.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { lightCategoricalColor, dictionaryToArray } from '@allenai/varnish';
 import { ChartingGrid, ColorValues } from './ColorValues';
 
 const lightCategoricalColorGroup = {
-  darkCategorical: dictionaryToArray(lightCategoricalColor),
+  lightCategorical: dictionaryToArray(lightCategoricalColor),
```
Comment (arnavic) [inline docs/data/material/varnish/colors/LightCategoricalColorValues.js:10]: learning question: why are we making the change here from dark to light categorical?

**Review by arnavic — APPROVED** [review]

File: docs/data/material/varnish/colors/LightCategoricalColorValues.js (hunk 4 above)
Comment (jonborchardt) [reply docs/data/material/varnish/colors/LightCategoricalColorValues.js:10]: this was a copy paste error when i added the colors earlier...
this should always have been light

**Pull request merged**